packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -145,6 +147,47 @@ describe('Atlas Login', function () {
       });
     });
 
+    describe('telemetry', () => {
+      let telemetry: Telemetry;
+
+      before(async function () {
+        telemetry = await startTelemetryServer();
+      });
+
+      it('should send identify after the user has logged in', async function () {
+        const atlasUserIdBefore = await browser.getFeature(
+          'telemetryAtlasUserId'
+        );
+        expect(atlasUserIdBefore).to.not.exist;
+
+        try {
+          await browser.openSettingsModal('Feature Preview');
+
+          await browser.clickVisible(Selectors.LogInWithAtlasButton);
+
+          const loginStatus = browser.$(Selectors.AtlasLoginStatus);
+          await browser.waitUntil(async () => {
+            return (
+              (await loginStatus.getText()).trim() ===
+              'Logged in with Atlas account test@example.com'
+            );
+          });
+        } finally {
+          await telemetry.stop();
```

Comment:
You can probably move this await telemetry.stop() to an after() hook since you start it in a before() hook. If the test fails before the try it will never run, for example.

Reply:
you're right, thank you!